arangod/RocksDBEngine/RocksDBAqlFunctions.cpp

```diff
@@ -389,7 +418,7 @@ void RocksDBAqlFunctions::registerResources() {
   TRI_ASSERT(functions != nullptr);
 
   // fulltext functions
-  functions->add({"FULLTEXT", ".h,.,.|.", false, true,
+  functions->add({"FULLTEXT", ".h,.,.|.", true, false,
```

Comment:
same comment as for MMFiles: `FULLTEXT` is not deterministic in the sense that results can be cached.